src/main/java/com/google/devtools/build/lib/starlarkbuildapi/platform/IncompatiblePlatformProviderApi.java

```diff
@@ -0,0 +1,11 @@
+package com.google.devtools.build.lib.starlarkbuildapi.platform;
+
+import com.google.devtools.build.docgen.annot.DocCategory;
+import net.starlark.java.annot.StarlarkBuiltin;
+import net.starlark.java.eval.StarlarkValue;
+
+@StarlarkBuiltin(
+    name = "IncompatiblePlatformProvider",
+    doc = "An interface for targets that are incompatible with the target platform.",
```

Comment:
Can we include a link to appropriate documentation to properly contextualize this?

Reply:
Good idea. I linked back to platforms.html in this case.